libc/optee/link.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This looks like duplicate code that could be shared across ELF64 architectures.

Reply:
Agreed; refactored so that `dl_iterate_phdr` is shared across OP-TEE and SGX.